.github/workflows/ci.yaml

```diff
@@ -13,6 +13,7 @@ jobs:
         os:
           - ubuntu-latest
           - macos-latest
+          - windows-latest
```

Comment:
![image](https://user-images.githubusercontent.com/13424970/172273825-edee22d5-ac76-40cd-9f15-b0e5aedf3682.png)

Dang, this now translates to 52 jobs per push to a non-fork PR. Did this take ~30m for everything to finish? Sounds like more reason to address nextstrain/private#48.

Reply:
Not sure exactly how long it takes given it depends on parallelism and runner availability (more acute with macOS), but yeah, it definitely takes longer now. :grimacing: That said, I think the most likely to fail tests run in the first few minutes, so that's should be an early indicator.

One quick change that I'd be inclined to make is to only run on `push` for `master` and otherwise only run on `pull_request`. This would slash total checks/jobs from 53 (52 + 1) → 27 (26 + 1).

Reply:
(The biggest downside of that is you could no longer push without opening a PR just to see what CI says, but I think that's an acceptable tradeoff, esp. with draft PRs now.)

Reply:
I'm in favor of that change. Especially since the limit is org-level, it'd be silly to have CI here block arguably more important things such as the [now-hourly monkeypox-ingest](https://github.com/nextstrain/monkeypox/blob/9c3f0b0faac63a94fd726c9174ed3456d78388b7/.github/workflows/fetch-and-ingest.yaml#L19).

Reply:
Sounds good. I'll add it to #180 since I'm adding the 3.10 stuff there anyway.

Reply:
Added there as https://github.com/nextstrain/cli/pull/180/commits/fab709a2f45fc74afe2a15038e877e4dd58ae222